tests/integration_tests/debezium_basic/run.sh

```
@@ -24,12 +24,25 @@ function run() {
 
 	run_cdc_server --workdir $WORK_DIR --binary $CDC_BINARY
 
-	SINK_URI="kafka://127.0.0.1:9092/$TOPIC_NAME?protocol=debezium&enable-tidb-extension=true"
+	if [ "$SINK_TYPE" == "kafka" ]; then
+		SINK_URI="kafka://127.0.0.1:9092/$TOPIC_NAME?protocol=debezium&enable-tidb-extension=true"
+	fi
+
+	if [ "$SINK_TYPE" == "pulsar" ]; then
+		run_pulsar_cluster $WORK_DIR normal
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)